plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -57,8 +57,7 @@ public void createStandardProcessor() throws IOException {
     public void testEnglishTextDocument() throws Exception {
         Map<String, Object> attachmentData = parseDocument("text-in-english.txt", processor);
 
```

Comment:
does it make sense to add some test for the new getFieldValueAsBytes (or maybe move some of this below) to IngestDocumentTests ?